### PR TITLE
Added getMaxTimestampError() to shared clock

### PR DESCRIPTION
**Pull request by Stevenic**
Lets each client compute their maximum timestamp error. Each client can then transmit this error rate to other clients to compute the total maximum clock drift between all clients.

### CODE REVIEW

**Review by ryanbliss — DISMISSED** [review]
Looks good to me, though perhaps the `SharedClock` could benefit from some brief comments / docs, like what is there for `getTimestamp`. Just a thought.

File: packages/live-share/src/LocalTimestampProvider.ts [hunk 1]
```diff
@@ -26,4 +26,8 @@ export class LocalTimestampProvider implements ITimestampProvider {
 
         return new Date().getTime();
     }
+
+    public getMaxTimestampError(): number {
+        return 0;
+    }
 }
```
Comment (huntj88) [inline packages/live-share/src/LocalTimestampProvider.ts:33]: missing new line?

**Review by huntj88 — APPROVED** [review]
Looks good

File: packages/live-share/src/LocalTimestampProvider.ts (hunk 1 above)
Comment (Stevenic) [reply packages/live-share/src/LocalTimestampProvider.ts:33]: before the return???

Comment (Stevenic) [reply packages/live-share/src/LocalTimestampProvider.ts:33]: Oh I see

Comment (Stevenic) [reply packages/live-share/src/LocalTimestampProvider.ts]: fixed

**Pull request merged**